packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -255,6 +255,11 @@ export type CallAdapterUiState = {
     page: CallCompositePage;
 };
 
+// Warning: (ae-internal-missing-underscore) The name "CallAgentCommon" should be prefixed with an underscore because the declaration is marked as @internal
+//
+// @internal (undocumented)
+export type CallAgentCommon = CallAgent;
```

Comment:
Why is this `@internal` type in the public API?

Same for `CallCommon`.

Reply:
Hmm, maybe I should make it public... this is not smart enough to make cast it back to CallAgent...